src/Microsoft.Health.Fhir.Stu3.Web/appsettings.json

```diff
@@ -23,7 +23,7 @@
         },
         "CoreFeatures": {
             "SupportsBatch": true,
-            "SupportsTransaction": false
+            "SupportsTransaction": true
```

Comment:
I think we should enable it with the transaction tests PR (in case if this PR gets merged before transaction tests PR). Thoughts?

Reply:
Reason I enabled this is to execute e2e test associated with it.  My plan is to wait for the PR on Transactione2etest to get merged before merging this to master.